src/Features/Core/SolutionCrawler/WorkCoordinator.NormalPriorityProcessor.cs

```diff
@@ -183,6 +187,35 @@ public Task Running
                         }
                     }
 
+                    private async Task WaitForHigherPriorityOperationsAsync()
```

Comment:
Can we refactor some of the common code here to the common base type between low and normal priority processor?